src/Ingredients/DisplayIngredients.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace BreadAndIfit\Ingredients;
+
+class DisplayIngredients
+{
+    /**
+     * This method calls the functions below and its purpose is to be called in the index.php to make
+     * the accordion of ingredients.
+     * @param array $ingredients is the array of objects that came from the database.
+     * @return string is the outputted accordion.
+     */
+    public static function displayIngredients(array $ingredients): string
+    {
+        $ingredientSorted = self::formatData($ingredients);
+
+        $output = self::outputHTML($ingredientSorted);
+
+        return $output;
+    }
+
+    /**
+     * This method formats the array of data into a multidimensional associative array of ingredients
+     * organised by their category.
+     * @param array $ingredients is the array of objects that came from the database.
+     * @return array is the formatted array.
+     */
+    private static function formatData(array $ingredients): array
+    {
+        $ingredientSorted = [];
+        foreach ($ingredients as $ingredient) {
+            $ingredientSorted[$ingredient->getCategory()][] = $ingredient->getName();
```

Comment:
this discards the entire ingredient object except name, while you may only need the name at the moment, this is not advisable. put the whole object into the array then use the name in the htmloutput function

Reply:
this also negates the purpose of using an object and means the names could be altered before being output accidentally.